### PR TITLE
main: Fix output file handling

### PR DESCRIPTION
**Pull request by sgreene570**
This will let you actually write an ignition file with the `--output` flag.
Also added a newline print to the error printing function.

### CODE REVIEW

File: internal/main.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ import (
 
 func fail(format string, args ...interface{}) {
 	fmt.Fprintf(os.Stderr, format, args...)
+	fmt.Println()
```
Comment (ajeddeloh) [inline internal/main.go]: this should be added to the fmt string where fail() is called instead.

Comment (ashcrow) [reply internal/main.go]: IE:

```golang
fmt.Fprintf(os.Stderr, fmt.Sprintf("%s\n", format), args...)
```

or added to each call of `fail()`'s format.

Comment (ajeddeloh) [reply internal/main.go]: added to each call of fail() I think.

Comment (sgreene570) [reply internal/main.go]: will do

**Comment by sgreene570** [comment]
should be good to go now.

**Comment by sgreene570** [comment]
Closes #12

**Pull request merged**